src/jmh/java/dev/blaauwendraad/masker/json/JsonMaskerBenchmark.java

```diff
@@ -66,4 +68,9 @@ public String jsonMaskerString(State state) {
     public byte[] jsonMaskerBytes(State state) {
         return state.jsonMasker.mask(state.jsonBytes);
     }
+
+    @Benchmark
+    public void jsonMaskerByteArrayStreams(State state) {
+        state.jsonMasker.mask(new ByteArrayInputStream(state.jsonBytes), new ByteArrayOutputStream());
+    }
```

Comment:
Isn't this also covered by the benchmark below?

Reply:
It is. I included streaming mode benchmarks here to show how it compares to the rest of the modes with varying masker parameters, while the idea of the benchmark below (`StreamTypeBenchmark`) is to see how streaming mode performs for different types of streams with fixed masker parameters.